src/metabolic_ninja/worker/designer.py

```diff
@@ -181,6 +191,25 @@ def evaluate_opt_gene(designs, pathway, model, method):
                     for g in design_result.targets
                     if isinstance(g, cameo.core.target.GeneKnockoutTarget)
                 }
+                gene_targets = {}
+                for t in list(knockouts):
```

Comment:
It seems superfluous to transform knockouts to a list? Also, please use at least three letter variable names outside of list comprehensions.

```suggestion
                for target in knockouts:
```